tools/wptrunner/wptrunner/testrunner.py

```diff
@@ -490,7 +490,7 @@ def start_test_runner(self):
                 self.child_stop_flag)
         self.test_runner_proc = Process(target=start_runner,
                                         args=args,
-                                        name="Thread-TestRunner-%i" % self.manager_number)
+                                        name="TestRunner-%i" % self.manager_number)
```

Comment:
I changed this first to "Process -TestRunner-%i" for reasons of pedantry, but then when I made the process raise an (uncaught) exception to see what would happen, what showed up in the log was "STDERR: Process Thread-TestRunner-1:". So maybe just no prefix, since the classes already say what kinds of things these are.